shared/test/test_files.rb

```diff
@@ -344,6 +344,38 @@ def test_bogus_metadata
     assert not_found?
   end
 
+  def test_rename_operations
```

Comment:
No, not really an overwrite - just a regular rename. Since you also added a rename text, I should probably rename this to be more explicit that this test is verifying rename with mixed case and spaces (mixed case was the bug that I was fixing, that wasn't covered by our tests)